src/connectors/connectorSwUpdates.js

```diff
@@ -70,10 +70,16 @@ export default class ConnectorSwUpdates extends Connector{
 
     subscribe = (input) =>
         new Promise((resolve, reject) => {
-            if (this.config.checkForUpdatesAtBoot){
-                this._checkForUpdates();
+            if (this.config.checkForUpdates) {
+                console.log("Software updates enabled");

```

Comment:
don't `console.log`

Reply:
The BGPAlerter binary output to stdout is capture for functional tests, so I need to use console.log(...). The the defined logger output to ./logs/reports... and ./log/error... are generated by FileLoggers defined in env.js